Scripts/SaneDocReport/SaneDocReport.py

```diff
@@ -10,13 +13,20 @@
 try:
     sane_json_b64 = demisto.args().get('sane_docx_report_base64', '').encode(
         "utf-8")
+    orientation = demisto.args().get('orientation', 'portrait').encode(
+        "utf-8")
+    paper_size = demisto.args().get('paperSize', 'A4').encode(
+        "utf-8")
     with open('sane.json', 'wb') as f:
         f.write(base64.b64decode(sane_json_b64))
 
-    run('sane.json', OUTPUT_FILE_PATH)
+    run('sane.json', OUTPUT_FILE_PATH, {
+        'orientation': orientation.decode('utf-8', 'ignore'),
+        'paper_size': paper_size.decode('utf-8', 'ignore')
+    })
 
     with open(OUTPUT_FILE_PATH, 'rb') as f:
-        encoded = base64.b64encode(f.read()).decode("utf-8", "ignore")
+        encoded = base64.b64encode(f.read()).decode('utf-8', 'ignore')
```

Comment:
lets stay consistent with `"` and `'` and use one of them (preferably `'`) in all the code